components/portal/widgets/controllers.js

```diff
@@ -421,7 +432,9 @@ define(['angular', 'moment'], function(angular, moment) {
         }
 
         // If this is the last time through the loop, turn off loading spinner
+        // and reorder the list by quantity
```

Comment:
I'm not sure on this ordering-by-quantity feature. I can imagine use cases where the items are semantically ordered. But maybe we don't have enough real world usages yet to get a feel for whether ordering-by-quantity is going to fly. Fine with trying it.